packages/react-components/src/components/styles/ControlBar.styles.ts

```diff
@@ -145,3 +145,7 @@ export const endCallControlButtonStyles: IButtonStyles = concatStyleSets(control
     color: palette.whiteTranslucent40
   }
 });
+
+export const defaultParticipantListContainerStyle: IStyle = {
+  maxHeight: '20rem'
```

Comment:
Is it possible this can be bounded to the window/available space instead of an arbitrary value? Just thinking if someone's window is somehow smaller than ~350px this will go beyond the screen height

Reply:
Actually ignore me, this is max height ~

Reply:
good one. Will fix that in next PR